GIBSDownloader/gibs_downloader.py

```diff
@@ -111,6 +112,8 @@ def main():
     parser.add_argument("--product", default=Product.viirs, type=Product, help="select the NASA imagery product", choices=list(Product))
     parser.add_argument("--keep-xml", default=False, type=bool, help="preserve the xml files generated to download images")
     parser.add_argument("--animate", default=False, type=bool, help="Generate a timelapse video of the downloaded region")
+    parser.add_argument("--name", default="MODIS Terra CorrectedReflectance TrueColor,0.25", type=str, help="enter the full name of the NASA imagery product and its image resolution separated by comma")
+    
 
```

Comment:
check if parser.product not None: 
   name = parser.product.get_name()
   res = .25
else: 
  name,res = ... as you have it now